core/lib/page-functions.js

```diff
@@ -78,6 +80,14 @@ function getElementsInDocument(selector) {
   return results;
 }
 
+/**
+ * @param {string} string
+ * @param {number} characterLimit
+ */
+function truncate(string, characterLimit) {
+  return Util.truncate(string, characterLimit);
```

Comment:
`Util` isn't injected into our execution context though. Shouldn't we be using `Util.truncate.toString()` somewhere?